core/dictionary.cpp

```diff
@@ -112,6 +112,15 @@ Variant Dictionary::get_valid(const Variant &p_key) const {
 	return E.get();
 }
 
+Variant Dictionary::get(const Variant &p_key, const Variant &p_default) const {
+	const Variant *result = getptr(p_key);
+	if (!result) {
+		return p_default;
+	}
+
+ 	return *result;
```

Comment:
You have a mix of tabs and spaces on this line. https://travis-ci.org/godotengine/godot/jobs/440229249#L536

Reply:
Thank you for finding that.  I just fixed it.